packages/mdc-chips/README.md

```diff
@@ -94,6 +94,7 @@ To customize the colors of any part of the chip, use the following mixins.
 
 Mixin | Description
 --- | ---
+`mdc-chip-set-margin($margin)` | Customizes the amount of space between each chip in the set
```

Comment:
RE the word "margin": I think the design folks have a specific word for "the amount of space between stuff", but I can't remember what it is... "Gutter" maybe? I would check with them

Reply:
I suspect gutter is more specific to e.g. layout grid and image list, things with a more consistent gap between rows/columns. Chips are inline and can wrap so I'm not sure it'd apply there, so better ask.

Reply:
Design suggested "padding" which I think makes more sense than "margin" but could be equally misleading. @acdvorak suggested "spacing", which sounds good to me.

Reply:
It sounds like design wants "padding", but both "padding" and "margin" have well-established meanings on the Web, so using either of those would likely confuse a lot of our clients.

What about "spacing"?